src/scss/_base.scss

```diff
@@ -31,12 +39,12 @@
 		text-align: right;
 	}
 
-	th[scope=row] {
-		border-right: 2px solid oColorsGetColorFor(o-table-header-row, border);
-	}
+	// th[scope=row] {
```

Comment:
Can these be fully removed instead of commented out? What's the reason for keeping them around?